src/Zikula/FormExtensionBundle/Form/Type/ControllerType.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
IMO this should be back in settings module. it is very specific to that usage and not a general purpose "ControllerType" validator. Someone may come along later and wonder why it doesn't work for that purpose.

Reply:
ah - it is not a validator - still the point remains valid. this is a very specific form type, not a general purpose. please move to SettingsModule

Reply:
This isn't a validator, but a form type. At least Content will use it, too. It may useful whereever a sub request should become configurable.